.github/workflows/semgrep2.yml

```diff
@@ -0,0 +1,18 @@
+name: Semgrep
+
+on:
+  pull_request: {}
+
+jobs:
+  semgrep:
+    name: Scan
+    runs-on: ubuntu-latest
+    if: (github.actor != 'dependabot[bot]')
```

Comment:
We should add whitesource here too https://github.com/snowflakedb/snowflake-connector-python/blob/9d7559bdab6ea972d69788d9197ae88dadb1f068/.github/workflows/jira_issue.yml#L10

Reply:
fixed